test/Interop/Cxx/foreign-reference/Inputs/inheritance.h

```diff
@@ -1,5 +1,12 @@
 #pragma once
 
+#ifndef TEST_INTEROP_CXX_FOREIGN_REFERENCE_INPUTS_INHERITANCE_H
+#define TEST_INTEROP_CXX_FOREIGN_REFERENCE_INPUTS_INHERITANCE_H
```

Comment:
This header already has a `#pragma once`, so the include guard is redundant. Could you please remove it?